go/parquet/pqarrow/encode_dictionary_test.go

```diff
@@ -400,7 +400,7 @@ func (ar *ArrowReadDictSuite) writeSimple() {
 		pqarrow.DefaultWriterProps()))
 }
 
-func (ArrowReadDictSuite) NullProbabilities() []float64 {
+func (*ArrowReadDictSuite) NullProbabilities() []float64 {
```

Comment:
Go asks to stick to one of the receivers type: either value or pointer (no mixing)